src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs

```diff
@@ -11,8 +11,8 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```

Comment:
`AssemblyVersion` remains the same, `AssemblyFileVersion` needs to be set to `4.0.0.0`

Reply:
Changed